Subtester/Assets/Plugins/Android/mainTemplate.gradle

```diff
@@ -0,0 +1,73 @@
+// GENERATED BY UNITY. REMOVE THIS COMMENT TO PREVENT OVERWRITING WHEN EXPORTING AGAIN
```

Comment:
could you elaborate on this? do we have to remember to remove the comment before exporting every time?

Reply:
This comment is automatic too, I didn't write it. We don't have to do anything, ED4MU should update the versions here automatically, but if it doesn't doing a Resolve should.